lib/angular/template/appform-field-location.tpl.html

```diff
@@ -22,6 +23,11 @@
       name="inputNameY"
       ng-model="fieldLocationCtrl.field.value.long"
       ng-required="fieldLocationCtrl.field.required"
+      ng-change="fieldLocationCtrl.updateModel()"
     >
   </md-input-container>
 </div>
+
+<div ng-show="fieldLocationCtrl.field.err">
+  Error: {{fieldLocationCtrl.field.err}}
+</div>
```

Comment:
Would be nice to have a css class for styling, and also keep this in the same div or `md-input-container` as the related field.